.github/workflows/build.yml

```diff
@@ -20,33 +20,34 @@ jobs:
     outputs:
       go-version: ${{ steps.get-go-version.outputs.go-version }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Determine Go version
         id: get-go-version
         run: |
           echo "Building with Go $(cat .go-version)"
-          echo "::set-output name=go-version::$(cat .go-version)"
+          echo "go-version=$(cat .go-version)" >> $GITHUB_OUTPUT
 
   get-product-version:
     runs-on: ubuntu-latest
     outputs:
       product-version: ${{ steps.get-product-version.outputs.product-version }}
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: get product version
         id: get-product-version
         run: |
           VERSION="${{ github.event.inputs.version || '0.0.0-dev' }}"
           echo "Using version ${VERSION}"
-          echo "::set-output name=product-version::${VERSION}"
+          echo "product-version=${VERSION}" >> $GITHUB_OUTPUT
```

Comment:
This is just some cleanup that gets rid of a couple set-output errors in the actions UI